### ISSUE TITLE
Deactivate/reactivate option

### ISSUE THREAD

**Issue by pirate486743186** [dillbyrne/random-agent-spoofer#458]
we should have an easy to deactivate/reactivate option.

**Comment by dillbyrne** [comment]
Their is one already in the mouse context menu and under the addon preferences in about:addons. In the current release there was a bug due to mozilla removing a preference. I ahve fixed the issue and will publish a new release shortly

**Comment by pirate486743186** [comment]
ahh, i haven't notice the one in the mouse menu.
I started using the addon some weeks ago, the current version only, and you have me confused now. What's the bug, and what's normal?

I tried to use the disable RAS in the mouse menu, and nothing seamed to happen, but it toggled "enable RAS" in its preference page. Also, if i click "real profile" the button gets gray only then (and the pop up says it's disabled too). I was expecting, that only the user agent was changed with that option. All this mess it's the bug? Activated/deactivated status is correctly displayed? It will get intuitive in the new version?

I really meant, an explicit option in the addon button on the toolbar. In other addons that you care about disabling, there is always the option in their button.
